src/app/tech-stack/components/TechStackCard.tsx

```diff
@@ -39,10 +32,23 @@ export default function TechStackCard({ title, data }: TechStackCardProps) {
                   />
                 ))}
               </div>
+              <Button
+                title={title}
+                customClassName="capitalize w-[62px] h-[32px] p-0 min-h-full text-xs font-medium text-base-300 bg-primary-content border-transparent mr-4 rounded-[32px] hover:bg-primary hover:border-transparent"
+              >
+                Vote
```

Comment:
I can't tell if this text is centered or not but it should be centered, if it isn't.